apps/zui/src/plugins/brimcap/zeek/queries.ts

```diff
@@ -25,6 +25,7 @@ export function findConnLog(pool: string, uid: string) {
   | (` +
     uidFilter(uid) +
     `)
+  | _path=="conn" 
```

Comment:
@jameskerr: I'll definitely appreciate your close review here. The function name `findConnLog()` and how I see it being used makes it pretty clear that we only ever want it to find true `conn` records, so I think adding what I've done here is safe and desirable. But please double check me!